### ISSUE TITLE
Icons grid size

### ISSUE THREAD

**Issue by adambelis** [friction2d/friction#181]
I ma tring to  fixe few icons i noticed that svg are on 25x25 grid .
this is slightly unusual  it usually goes 16 32 64 alternatively 16 24 32 48 68 128
Is this a mistake in files or intended and should be kept same ?

**Comment by rodlie** [comment]
What are you trying to fix?

The original icons from enve are random, when I need new icons or replace something I use https://github.com/Shrinks99/blender-icons

Note that Friction does not use the SVG's, they are just source for our icon theme.

**Comment by adambelis** [comment]
circle and  rectangles are way off in size (i know circle should be slightly bigger, but this is way too much).

> Note that Friction does not use the SVG's, they are just source for our icon theme.

Is that autogenerateed or that needs to be done by hand ?

**Comment by rodlie** [comment]
Ok, hold on doing anything yet, can be a good excuse to replace some old icons ;)

> Is that autogenerateed or that needs to be done by hand ?

We have a script that generates/update the icon theme from the SVG's (uses inkscape).

**Comment by adambelis** [comment]
![image](https://github.com/friction2d/friction/assets/50795097/491442e2-f647-4345-8b3e-5daf3a13b1ba)

 well thtas kind of rabbithole i strted falling to . Fricon does not have that many icons copper to inkscape +450 :D

**Comment by adambelis** [comment]
Yes, all Inkscape icons are CC . But Inkscape has more icon sets. Hicolor old with colors (depricated) Hicolor symbolic (single color) Multicolor (currently maintained mostly by me) tango (colorful depricated i am pushing for removing it soon ) dash (new single color theme).
defoult multicolo theme :
![image](https://github.com/friction2d/friction/assets/50795097/bb3a9b66-214b-4a3c-bca6-9e4fb1b5dc55)

Not sure if this will work for fricon since  this icons are design in specific way it has 3+1 color  every color is used contextually.
(pink = type, procedural shape , Green create new , blue-style color ) and this color case controlobale by CSS from UI
https://wiki.inkscape.org/wiki/Creating_a_multicolor_icon

bascaly its a mess :D 

https://gitlab.com/inkscape/inkscape/-/tree/1.4.x/share/icons

sometimes you can get away with using icons from other software but  you need to be carful if it fit you context and rest of icons . Blender is kind of crazy coz lots of the icons (not all) are made in bleder and are not svgs

**Comment by rodlie** [comment]
I started a new repo for the v1 icon theme: https://github.com/friction2d/friction-icon-theme

**Comment by adambelis** [comment]
should i  try   commit there ?

**Comment by rodlie** [comment]
Yes, that would be nice, we will use this theme for 0.9.6 (I added the old icons to avoid breaking anything, see hicolor/scalable/legacy folder).

Just put the icons in the hicolor/scalable/actions folder.
